scan/peers.py

```diff
@@ -65,7 +65,7 @@ def get_country_by_ip(ip: str) -> str:
 class PeerMonitorForm(forms.ModelForm):
     class Meta:
         model = PeerMonitor
-        fields = "__all__"
+        fields = ['announced_address', 'platform', 'application', 'version', 'height', 'cumulative_difficulty', 'country_code', 'state', 'downtime', 'lifetime', 'availability', 'last_online_at']
```

Comment:
Is this really needed?

Reply:
__all__ tries to write to the two reward columns too..  it overwrites the time field every time peers.py runs...   it was the only way i could allow a new node to create a record that was null without overwriting the null every time peer.py ran..  had to omit them from the form.